locales/en/index.yml

```diff
@@ -464,6 +464,7 @@ navigation:
   messages: Messages
   profile: Profile
   accessibility: "Section {{section}}, panel, {{order}} of 4"
+  status: Selected
```

Comment:
```suggestion
  selected: Selected
```
As commented by @pietro909, what you think about renaming `status` to `selected`, in order to have a better match with the meaning?

Reply:
Agree! I do it right away